src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,11 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Checks if this command mutates data in address book
+     *
+     * @return true if this command mutates data
+     */
```

Comment:
Good job remembering to add a header comment to this method. 👍 
It can be shortened as follows without losing any info. 
```java
/**
  * Returns true if the command potentially mutates data
  */
```